cdap-spark-core-base/src/main/java/io/cdap/cdap/app/runtime/spark/SparkProgramRuntimeProvider.java

```diff
@@ -211,11 +222,11 @@ private ProgramRunner createSparkProgramRunner(Injector injector,
                                                  String programRunnerClassName,
                                                  ClassLoader classLoader) {
     try {
-      ClassLoader oldClassLoader = ClassLoaders.setContextClassLoader(classLoader);
+      ClassLoader oldClassLoader = setContextClassLoader(classLoader);
```

Comment:
Seems like an unnecessary change? Typically we don't use static import.

Reply:
I think this was unintentional on my part; thanks for catching, fixed.